core/crypto/crypto_test.go

```diff
@@ -59,6 +61,9 @@ var (
 	queryTxCreators   []createTxFunc
 
 	ksPwd = []byte("This is a very very very long pw")
+	
+	attributes = map[string]string{"company":"IBM", "position":"Software Engineer"}
```

Comment:
Thanks @muralisrini your right unit test are running successfully in lasted master and in the abac branch too.